### PR TITLE
clh: update to latest master

### PR DESCRIPTION
**Pull request by jcvenegas**
Update to latest master code.

Fixes: #2343

Signed-off-by: Jose Carlos Venegas Munoz <jose.carlos.venegas.munoz@intel.com>

### CODE REVIEW

**Comment by sboeuf** [comment]
@jcvenegas please rebase on latest (latest) master from CH :)
We merged some new code today, and we added the OpenAPITools generation validation in our CI.
Two main things changed which will affect this PR:
- `CpuConfig` has been changed into `CpusConfig`
- `FsConfig` has a boolean field `dax`, which will allow you to explicitly say if you want DAX to be enabled or not.

File: versions.yaml [hunk 1]
```diff
@@ -75,7 +75,7 @@ assets:
       url: "https://github.com/intel/cloud-hypervisor"
       uscan-url: >-
         https://github.com/intel/cloud-hypervisor/tags.*/v?(\d\S+)\.tar\.gz
-      version: "ca97385da5ebb798"
+      version: "ee528ae80836e9d59ff50d124ae356ed1b07002a"
```
Comment (jodh-intel) [inline versions.yaml]: I wonder if we should treat this as a re-vendor? If so, would be good to include the changes between these commits in one of the commits in this PR.

Comment (jcvenegas) [reply versions.yaml]: good idea, added the changes in the commit

**Pull request merged**